TheJumpCloud/support
# CUT-4428: Forget RADIUS WiFi Network v2.0

## Issues
* [CUT-4428](https://jumpcloud.atlassian.net/browse/CUT-4428) - Forget RADIUS WiFi Network v2.0

## What does this solve?
Updated command to include a new prompt for the end user letting them know that they will need to re-enter their credentials to the specified network

## Is there anything particularly tricky?
N/A

## How should this be tested?

1. Import the command into your organization
2. Assign a RADIUS device to the command
3. Change the user's password
4. Ensure that the scheduled task runs to forget the network and the new prompt appears
5. When hitting OK on the prompt, the network details pane should open

## Screenshots
![image](https://github.com/user-attachments/assets/d7b693ca-d62c-4fa0-a84d-40b8c371c4e3)
![image](https://github.com/user-attachments/assets/7231cab4-71f2-4046-ab5c-720fb66eec1c)


[CUT-4428]: https://jumpcloud.atlassian.net/browse/CUT-4428?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
